src/jdk.httpserver/share/classes/com/sun/net/httpserver/Authenticator.java

```diff
@@ -132,7 +130,7 @@ public Retry (int responseCode) {
         /**
          * returns the response code to send to the client
```

Comment:
`* Returns the response code to send to the client.`

Reply:
Change made in https://github.com/openjdk/jdk/pull/301/commits/604ea45ca36c8fb0cb195c581f822dc5ae1c23c1